app/views/prx_auth/rails/sessions/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<script type='application/javascript'>
+
+  function launchFlow(){
+    const idUrl = '<%= @id_host %>';
+    const idParams = '<%= @id_auth_params.html_safe %>';
+
+    window.location = '//' + idUrl + '/authorize?' + idParams;
```

Comment:
Yeah, after reading this all through ... i _think_ the only diffs with exchange are:

1. First redirect-to-ID in JS vs a 302
2. Exchange actually uses a `<form>` to POST the token, so it can auto-follow the 302.

I'm not sure either of those _really_ matter.  Though maybe they're a bit more secure from a server-logs perspective?  (Your token isn't sent via a query param, which maybe the server is accidentally logging somewhere).

Thoughts?